src/cljs/rems/fields.cljs

```diff
@@ -179,13 +217,14 @@
         value (:field/value opts)
         options (:field/options opts)
         selected-keys (decode-option-keys value)]
-    ;; TODO: for accessibility these checkboxes would be best wrapped in a fieldset
     [field-wrapper
-     (assoc opts :readonly-component [readonly-field {:id (id-to-name id)
-                                                      :value (->> options
-                                                                  (filter #(contains? selected-keys (:key %)))
-                                                                  (map #(localized (:label %)))
-                                                                  (str/join ", "))}])
+     (assoc opts
+            :fieldset true
```

Comment:
could fieldset be always true?

Reply:
Fieldset is meant for grouping together multiple labels and their inputs. It doesn't make much sense for individual inputs - the fieldset legend and the field label would need to repeat the same text. The `field-wrapper` component needs to be refactored. Rich Hickey would say it's too complex.